### PR TITLE
Update README.md to remove language about legacy build systems.

### PR DESCRIPTION
**Pull request by JonET**
Ember-cli is great, but some of us still like using our existing build systems that have been polished over time and are still being developed. I think making add-ons available for them is a fantastic idea though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This is a work-in-progress. It's alpha quality. Feedback appreciated.
 # Why?
 
 The Ember addons ecosystem is one of the best things about Ember. But
-users stuck on legacy build systems have a hard time taking advantage
+users of non ember-cli build systems have a hard time taking advantage
```
Comment (jayphelps) [inline README.md:8]: This is grammatically strange to me. How bout something like:

> But if you're not using [ember-cli](http://www.ember-cli.com/) yet, taking advantage of them has historically been tough.

Comment (JonET) [inline README.md:8]: I'd just want to remove the word "yet". I wanted to take away the implication that not using ember-cli was somehow incorrect.

> But if you're not using ember-cli, taking advantage of them has historically been tough.

...maybe?

Comment (jayphelps) [inline README.md:8]: I don't know if there has been an "official" word on that, but from what I've seen it definitely won't be a priority going forward. Idiomatic, no; ember-cli is the prescribed way going forward. Incorrect? I think too strong of word to know at this point.

Comment (jayphelps) [inline README.md:8]: Take my opinion as just that, though. I'm not on the core team.

**Comment by ef4** [comment]
I appreciate the feedback, but anything other than ember-cli really is "legacy", in the sense that it's fine for existing applications to keep doing what they're doing, but _new_ applications absolutely should use ember-cli.

I expect many people to keep using other build systems because that's what they already have. And that's totally fine -- I wrote this addon specifically to help those people.

But I also don't want newbies to have any doubt in their minds: they should be using ember-cli.

**Pull request closed without merging**